include/deal.II/distributed/tria.h

```diff
@@ -633,179 +652,9 @@ namespace parallel
        * set to false, the triangulation is only repartitioned if needed (i.e.
```

Comment:
DistributedTriangulationBase::register_data_attach() 
DistributedTriangulationBase::notify_ready_to_unpack()

in line 646